ctapipe/io/tests/test_hdf5.py

```diff
@@ -804,3 +804,53 @@ def test_write_default_container(cls, tmp_path):
                 pytest.xfail()
             else:
                 raise
+
+
+def test_strings(tmp_path):
+    '''Test we can write unicode strings'''
+    from ctapipe.core import Container
+    from ctapipe.io import read_table
+
+    # when not giving a max_len, should be taken from the first container
+    class Container(Container):
```

Comment:
maybe not a good idea to call it Container (shadowing it's parent)

Reply:
Only inside this test function, but yes